src/core/webidl-index.js

```diff
@@ -42,8 +42,11 @@ export function run(conf, doc, cb) {
   Array.from(document.querySelectorAll("pre.def.idl"))
     .map(elem => {
       const span = document.createElement("span");
-      const clone = elem.cloneNode(true).firstElementChild;
-      span.appendChild(clone);
+      const clone = elem.cloneNode(true).children;
+      Array.from(clone).forEach(child => {
+        span.appendChild(child);
+        span.appendChild(document.createTextNode("\n"));
```

Comment:
If a child is already a new line, should we won’t this give us multiple new lines?

Reply:
Yes. This is giving an extra newline after every ```pre.idl```. Looking into this.